apps/demo-angular/src/plugin-demos/wcg-url.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, NgZone } from '@angular/core';
+import { DemoSharedWcgUrl } from '@demo/shared';
+import {} from '@nativescript/wcg-url';
```

Comment:
Is this an import for effects, or unneeded?
```suggestion
import '@nativescript/wcg-url';
```